src/views/index.nunjucks

```diff
@@ -40,5 +40,6 @@
   </div>
 </div>
 
+<script>const YODASPEAK_TRANSLATE_ENDPOINT = '{{ translationEndpoint }}';</script>
```

Comment:
Does this have to be here? Could it just be `src/public/js/index.js` file?

Reply:
This was the easiest way to render the config from the backend in order for clientside JS to access it.

What if I changed index.js to have an init function which takes this translation endpoint, then I just pass the value to that. This would eliminate this extra script element.

Reply:
Can we otherwise leave out the custom endpoint? It sounds a bit confusing to me, why this would be variable?

Reply:
Maybe if we want to show off a variable on the frontend we can do some configuration of the color of the button or something like that?

Reply:
It's only for clientside building, so a static site could point to any endpoint, e.g., if it was deployed to Netlify. When running with Express, it defaults to `/translate`.

Reply:
Gotcha, I guess it's fine like this then 👍